ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -268,6 +276,13 @@ class FamilyReads extends React.PureComponent {
     locus: null,
   }
 
+  updateReads = (familyGuid, locus, sampleTypes, tissueType) => {
+    const rnaReferenceOptions = [NORM_GTEX_TRACK_OPTIONS, AGG_GTEX_TRACK_OPTIONS].map(options => (
+      options.find(opt => opt.value.name.includes(TISSUE_REFERENCE_KEY[tissueType]))?.value
```

Comment:
you should construct a constant for a dictionary to look this up, instead of doing this `find` every time its instantiated, so you can just do `rnaReferenceOptions = RNA_TISSUE_REFERENCES[tissueType]`